lib/renderer/web-view/web-view.js

```diff
@@ -379,7 +379,8 @@ var registerWebViewElement = function () {
     'downloadURL',
     'inspectServiceWorker',
     'print',
-    'printToPDF'
+    'printToPDF',
+    'showDefinitionForSelection'
```

Comment:
@zcbenz do you think we should still be adding new APIs to `<webview>` that just pass through to `webContents` APIs or should this be accessible only via  `<webview>.getWebContents(). showDefinitionForSelection()`?